### PR TITLE
Update tomcat Image

### PR DESCRIPTION
**Pull request by KrisIliev**
### What does this PR do?
Updates tomcat Image to version 9. Fixes Helm security issues.

1. Image: tomcat:9.0.56-jre11-openjdk 

### What issues does this PR fix or reference?
**#892** 

#### Release Notes

**6.1.18**

### CODE REVIEW

File: releng/Dockerfile-tomcat [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Docker descriptor for Dirigible
 # License - http://www.eclipse.org/legal/epl-v20.html
  
-FROM tomcat:8.5.43-jdk11-openjdk
+FROM tomcat:9.0.56-jre11-openjdk 
  
-RUN rm -R /usr/local/tomcat/webapps/*
```
[the remark this answers is not in the document]
Comment (krasimirdermendzhiev) [reply releng/Dockerfile-tomcat:6]: In new Docker image tomcat:9.0.56-jre11-openjdk pre-packaged are removed.

File: releng/anonymous-runtime/Dockerfile-base [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # Docker descriptor for Dirigible
 # License - http://www.eclipse.org/legal/epl-v20.html
 
-FROM tomcat:8.5.43-jdk11-openjdk
+FROM tomcat:9.0.56-jre11-openjdk 
 
-RUN rm -R /usr/local/tomcat/webapps/*
```
[the remark this answers is not in the document]
Comment (krasimirdermendzhiev) [reply releng/anonymous-runtime/Dockerfile-base:6]: In new Docker image tomcat:9.0.56-jre11-openjdk pre-packaged are removed.

**Review by krasimirdermendzhiev — COMMENTED** [review]
In new Docker image tomcat:9.0.56-jre11-openjdk pre-packaged are removed. That's why the line
RUN rm -R /usr/local/tomcat/webapps/* is removed, but COPY target/ROOT.war $CATALINA_HOME/webapps/ shoud be there. White space will be removed with new commit and the test docker file will be remove it.

**Review by ThuF — APPROVED** [review]
+1 approved, @KrisIliev waiting for a new commit to delete the blank space after the at the end of **FROM tomcat:9.0.56-jre11-openjdk` `**

**Pull request merged**